app/pages/app/app.js

```diff
@@ -343,6 +344,26 @@ export class AppComponent extends React.Component {
     return null;
   }
 
+  renderNavPatientHeader() {
+    const { 
+      patient, 
+      userIsCurrentPatient,
+      permsOfLoggedInUser,
+      context: { trackMetric }
+    } = this.props;
+
+    if (!patient) return null; // only show on pages with a patient of focus
```

Comment:
This one line is doing a deceptive amount of heavy lifting here.

Looking at `app/components/navbar/navbar.js:144` which defined whether the `<NavbarPatientCard />` was rendered, it _looks like_ we rendered it as long as there is a value for `patient`. So I think this condition is in line with the previous one.